guides/troubleshooting/inotify-watch-limits.md

```diff
@@ -64,12 +61,17 @@ Because these settings are not namespace-aware, the values will be the same
 regardless of whether you run the commands on the host system or inside a
 container running on that host.
 
+> See [inotify(7)](https://man7.org/linux/man-pages/man7/inotify.7.html) for
+additional details regarding the `inotify` system.
```

Comment:
I understand this is used to indicate an aside visually, but semantically, I don't think this is correct because it puts the section in a [`<blockquote>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote) and this isn't quoting someone else

It seems alright, though? :man_shrugging: 

I do think we should style each line with a `>` prefix, this syntax is meant to replicate how [email replies typically quote people](https://en.wikipedia.org/wiki/Posting_style#Quoted_line_prefix) in plain text:

```markdown
> See [inotify(7)](https://man7.org/linux/man-pages/man7/inotify.7.html) for
> additional details regarding the `inotify` system.
```

Reply:
That's a valid point. What we need is a variety of banners and such, but until then, I think this is fine even if, as you point out, it's semantically incorrect.